app/schemas/models/course_instance.schema.coffee

```diff
@@ -19,6 +19,7 @@ _.extend CourseInstanceSchema.properties,
     language: {type: 'string', 'enum': ['python', 'javascript']}
   hourOfCode: { type: 'boolean', description: 'Deprecated, do not use.' }
   stats: c.object({ additionalProperties: true })
+  startLockedLevel: {type: 'string'}
```

Comment:
Should this be more specific and use a [`c.shortString`](https://github.com/codecombat/codecombat/blob/a1f05a2279d39e0a05b4941bfabc02cf6b1a7372/app/schemas/schemas.coffee#L18)?